src/pages/CommitDetailPage/CommitCoverage/CommitCoverage.jsx

```diff
@@ -98,16 +98,16 @@ function CommitRoutes() {
 function CommitErrorBanners() {
   const { owner } = useParams()
   const { data: ownerData } = useOwner({ username: owner })
-  const { data: commitErrorData } = useCommitErrors()
+  const { data } = useCommitErrors()
```

Comment:
nit: sometimes it's good to rename data returned if we're calling more than one query in a file